src/transformers/configuration_utils.py

```diff
@@ -163,7 +163,7 @@ class PretrainedConfig(object):
 
     def __init__(self, **kwargs):
         # Attributes with defaults
-        self.return_dict = kwargs.pop("return_dict", False)
+        self.return_dict = kwargs.pop("return_dict", True)
```

Comment:
Can we do this in another PR maybe -> this is breaking and affects all PT models as well

Reply:
All the tests pass no?

Reply:
All tests are passing is not an excuse for an instant breaking change on the whole library. There may be releases between now and the v4. This change can only be made once we go to v4. In the meantime, the switch can be activated:
- by ignoring the config value in TFBertMainLayer and just storing `True`. This removes the possibility of setting it to False via the config momentarily, but it can still be passed in the forward in eager mode.
- in all TF models by adding this line in the init of `TFPreTrainedModel`.

To first merge this PR without too many changes, I would favor using the first solution for now, then switch to all TF models later on when they are updated, and only do the global switch on the eve of the v4 release.